### ISSUE TITLE
[Unstable] SMG magazine recycling recipe requires a full magazine

### ISSUE THREAD

**Issue by BlueTheKing** [Regalis11/Barotrauma#8185]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
The SMG magazine recycling recipe requires a full magazine instead of any empty one.
```
<Fabricate suitablefabricators="fabricator" requiredtime="15" displayname="recycleitem">
    <RequiredSkill identifier="weapons" level="40" />
    <RequiredItem identifier="magnesium" />
    <RequiredItem identifier="lead" />
    <RequiredItem identifier="smgmagazine" />
</Fabricate>
```

**Version**
0.16.2.0

**Comment by ekqvist** [comment]
Fixed in Regalis11/Barotrauma-development@fffe958

**Comment by Rokvach** [comment]
Tested, no issues. Closing.
